src/js/themes/base.js

```diff
@@ -487,19 +487,33 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           },
         },
       },
+      content: {
```

Comment:
I think we should add a comment above `formField` indicating that these are setting grommet properties, not CSS directly.

Reply:
hum. i can definitely add a comment for that. but our theme object never assumed it was css only. take Select as an example, we use the theme to allow a custom component to use for the TextInput. we have other examples.

Although I agree most of the theme entries affect css directly, it is not 100% the case. So, I'm not sure if we should add a comment for all exceptions there.